docs/users/deleting-users.mdx

```diff
@@ -1,5 +1,69 @@
 ---
-sanity_slug: /docs/users/deleting-users
+title: Deleting users
+description: Learn how to delete users from your Clerk backend.
 ---
 
-# This is a stub
+# Deleting users
+
+Clerk currently supports deleting users through our [backend API](https://clerk.com/docs/reference/backend-api/tag/Users#operation/DeleteUser). 
+
+The [`deleteUser()`](/docs/references/backend/user/delete-user) method takes a single argument: the user ID of the user you want to delete. It can be accessed from the `users` sub-api of the `clerkClient`. 
```

Comment:
```suggestion
The [`deleteUser()`](/docs/references/backend/user/delete-user) method takes a single argument: the user ID of the user you want to delete. It can be accessed from the `users` sub-api of the `clerkClient` instance.
```